packages/site/pages/components/searchinput.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+
+import SearchInput from '@pluralsight/ps-design-system-searchinput/react'
+import Text from '@pluralsight/ps-design-system-text/react'
+
+import {
+  Chrome,
+  Code,
+  Content,
+  Example,
+  P,
+  PageHeading,
+  PropTypes,
+  SectionHeading,
+  withServerProps
+} from '../../src/ui'
+
+export default withServerProps(_ => (
+  <Chrome>
+    <Content title="Search Input">
+      <PageHeading packageName="searchinput">Search Input</PageHeading>
+
+      <P>Install the component dependency:</P>
+      <Code language="bash">
+        npm install @pluralsight/ps-design-system-searchinput
+      </Code>
+
+      <P>Include a React component in your project:</P>
+      <Code language="javascript">
+        import SearchInput from
+        '@pluralsight/ps-design-system-searchinput/react'
+      </Code>
+
+      <PropTypes
+        props={[
+          PropTypes.row([
+            'onClear',
+            'function',
+            null,
+            null,
+            'show clear button and trigger when button is clicked'
+          ]),
+          PropTypes.row([
+            'loading',
+            'boolean',
+            null,
+            false,
+            'show progress indicator'
+          ])
+        ]}
+      />
+      <P>
+        The normal React form-related props are also acceptable and expected.
+      </P>
+
+      <SectionHeading>Example</SectionHeading>
+      <P>
+        The <Text.Code>SearchInput</Text.Code> component can be just like the{' '}
```

Comment:
Would probably adjust this paragraph since some will be disabled, ie label and subLabel.

In fact, I might just explicitly show the props available in the proptypes table above